app/controllers/api/v1/bracks_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Api::V1::BracksController < ApplicationController
+
+  def index
+    latlng = params[:latlng]
```

Comment:
if you are just using this variable on the next line, no need to set it as a local variable 👍